code/src/components/DropDown.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+export const DropDown = (props) => {
+  const { age, setAge } = props
+
+  return (
+    <div className="question" tabIndex="0">
+      What is your age?
+      <select
+        onChange={event => setAge(event.target.value)} 
+        value={age}>
+        <option 
+          value="" 
+          className="answerOption">
```

Comment:
We are not able to style `<option />` tags, therefore there are no reasons to add class here

Reply:
removed these. :)